File: q/c/submissions/CYBERPHASE/problem1.java
class problem1
{
public static void main(int n)
{
int i=0,c=1;
for(i=1;i<=n;i++)
{
if(n%i==0)
{
c=c+1;
}
}
if(c==2)
{System.out.println("Yes");
}
else
{
System.out.println("No");
}
}
}

File: q/c/submissions/pro/problem1.cpp
#include<iostream>
using namespace std;
class problem1
{
void main()
{ 
long long n;
cout<<"enter n";
cin>>n;
int c=0;
for(long long i=0;i<=n;i++)
{
if(n%i==0)
c++;
}
if(c==2)
cout<<"YES";
else
cout<<"NO";
}
};


